Added information about copying blocks (#325)
ibexa/documentation-user: docs/content_management/create_edit_pages.md

- Move up block - allows you to change position of the block on the page by moving it up
- Move down block - allows you to change position of the block on the page by moving it down
- Block settings - allows you to access properties window
- Copy - allows you to reuse a configured block across pages by using the Clipboard section.
- Duplicate - duplicates existing block with its settings
- Refresh - refreshes preview of the block
- Delete - deletes existing block

![Block window - size adjustment](img/block_window_size_adjust.png "Block window - size adjustment")

#### Copy blocks

> Copying blocks allows you to reuse configured blocks across different pages.

Reviewer: ```suggestion
You can copy blocks to reuse them on other pages.
```
I'd focus on the reader, not the ssystem.

Reviewer 2: nitpick: you can use it on the same page as well.

> When you execute the "Copy" action on a block, it creates a new entry in the "Clipboard" section of the Page Blocks toolbox. Inserting the block from the clipboard adds an exact replica of the block, including its configuration, as it existed at the time of copying.

Reviewer: ```suggestion
When you copy a block, it gets saved in the "Clipboard" section of the Page Blocks toolbox.
Inserting the block from the clipboard puts an exact copy of it, with all its settings, as it was when you copied it.
```
How exactly do you insert a block? I'd focus on the steps, not the behavior of the system, something along the lines of:
"To do it, in the block's toolbar, click Copy. Then, in the Page Blocks toolbox, copy the content of the Clipboard section. You can now switch to another page and insert the block by..."

Author: You can drag and drop the block from the clipboard section to the page - exactly like the other blocks (the only difference is that it's inserted preconfigured).

> The clipboard can store only one block at a time. The content of the clipboard remains accessible while editing pages and is visible only to you.

Reviewer: ```suggestion
The clipboard can store only one block at a time.
The content of the clipboard remains accessible while editing pages and is visible only to you.
```
"accessible while editing pages " does it persist closing one page and opening another for editing?

Author: Yes, exactly, that's the main use case - for "single page editing" the Duplicate action is much faster, copying is for copying a block across different pages.

- Move up block - allows you to change position of the block on the page by moving it up
- Move down block - allows you to change position of the block on the page by moving it down
- Block settings - allows you to access properties window
- Copy - allows you to reuse a configured block across pages by using the Clipboard section.
- Duplicate - duplicates existing block with its settings
- Refresh - refreshes preview of the block
- Delete - deletes existing block

![Block window - size adjustment](img/block_window_size_adjust.png "Block window - size adjustment")

#### Copy blocks

You can copy blocks to reuse them on other pages.

> When you copy a block, it gets saved in the "Clipboard" section of the Page Blocks toolbox.

Reviewer: ```suggestion
When you copy a block, it gets saved in the "Clipboard" section of the Page Blocks toolbox.
I assume at one point this must happen?
"At this point you can continue working in the current page and you can close it and open another one, where you want to reuse the block"?

Author: Added as:
```
At this point you can continue working in the current page, or you can close it and open another one, where you want to reuse the block.
```

> From there you can drag and drop the block to the page like any other block.

Reviewer: ```suggestion
You can drag and drop the block from clipboard to a drop zone, like any other block.
```

Doing so puts an exact copy of the block on the page, with all its settings, as it was when you copied it.

The clipboard can store only one block at a time.

> The content of the clipboard remains accessible while editing different pages and is visible only to you.

Reviewer: ```suggestion
Contents of the clipboard are visible only to you, and...
```
"preserve switching into a new session"?
"remain accessible until the end of session"?

Author: I've chosen `preserve switching into a new session`
